tests/pytests/test_expire.py

```diff
@@ -126,11 +133,11 @@ def expireDocs(env, isSortable, expected_results):
 
 
         # test with WITHSCORES and EXPLAINSCORE - make sure all memory is released
-        conn.execute_command('HSET', 'doc1', 't', 'zoo')
+        conn.execute_command('HSET', 'doc1', 't', 'bar')
 
         # both docs exist
         expected_res = [2, 'doc2', res_score_and_explanation, ['t', 'foo'],
-                           'doc1', res_score_and_explanation, ['t', 'zoo']]
+                           'doc1', res_score_and_explanation, ['t', 'bar']]
```

Comment:
what was wrong with zoo :(

Reply:
Because we now return partial results, it was simpler to recreate the document with the same value and use the same expected result